data/remnant/remnant ships.txt

```diff
@@ -230,6 +230,324 @@ ship "Albatross" "Albatross (Ember Shield)"
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		weapon
+			"blast radius" 260
+			"shield damage" 2600
+			"hull damage" 1300
+			"hit force" 3900
+		"automaton" 1
+		inscrutable 1
+		"quantum keystone" 1
+		"burn protection" 100
+		"corrosion protection" 100
+		"discharge protection" 1
+		"disruption protection" 100
+		"force protection" 100
+		"fuel protection" 100
+		"hull protection" 100
+		"ion protection" 1
+		"leak protection" 100
+		"piercing protection" 100
+		"shield protection" 100
+		"slowing protection" 1
+	outfits
+		"Fusion Reactor"
+		"LP144a Battery Pack"
+		"D94-YV Shield Generator"
+		"Large Radar Jammer"
+		"Ramscoop"
+		"NDR-114 Android" 100
+		"Laser Rifle" 100
+		
+		"X4700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Hyperdrive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	gun 65 -111 
+		angle 40
+	gun -72 -97.5 
+		angle 320
+	gun -66.5 61 
+		angle 270
+	gun 78 64 
+		angle 90
+	turret 10 -136.5
+	turret -21.5 -57.5
+	turret 51 101
+	turret -63.5 120
+	turret 52.5 143
+		
+
+ship "Modified Dromedary" "Modified Dromedary Specter"
+	sprite "ship/modified dromedary specter"
+	thumbnail "thumbnail/modified dromedary wreck"
+	"uncapturable"
+	"never disabled"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 57600000
+		"shields" 5100
+		"hull" 5600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
+		"drag" 16.1
+		"heat dissipation" .6
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		weapon
+			"blast radius" 260
+			"shield damage" 2600
+			"hull damage" 1300
+			"hit force" 3900
+		"automaton" 1
+		inscrutable 1
+		"quantum keystone" 1
+		"burn protection" 100
+		"corrosion protection" 100
+		"discharge protection" 1
+		"disruption protection" 100
+		"force protection" 100
+		"fuel protection" 100
+		"hull protection" 100
+		"ion protection" 1
+		"leak protection" 100
+		"piercing protection" 100
+		"shield protection" 100
+		"slowing protection" 1
+	outfits
+		"Fusion Reactor"
+		"LP144a Battery Pack"
+		"D94-YV Shield Generator"
+		"Large Radar Jammer"
+		"Ramscoop"
+		"NDR-114 Android" 100
+		"Laser Rifle" 15
+		
+		"X4700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Hyperdrive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	gun 65 -111 
+		angle 40
+	gun -72 -97.5 
+		angle 320
+	gun -66.5 61 
+		angle 270
+	gun 78 64 
+		angle 90
+	turret 10 -136.5
+	turret -21.5 -57.5
+	turret 51 101
+	turret -63.5 120
+	turret 52.5 143
+
+
+ship "Modified Dromedary" "Modified Dromedary Wreck"
+	sprite "ship/modified dromedary wreck"
+	thumbnail "thumbnail/modified dromedary wreck"
```

Comment:
Do we want to make this variant never disabled as well? The only time is shows up is as a friendly NPC, but technically there's nothing stopping the player from stealing it, giving them a ship with all these resistances.

Reply:
Yes, definitely.

Reply:
It is can no longer be disabled or captured. Lore reason being that it is basically running in on emergency systems, so taking enough damage to stop it again would just finish it off entirely rather than disable it cleanly.